slick/src/main/scala/akka/stream/alpakka/slick/scaladsl/Slick.scala

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.slick.scaladsl
+
+import scala.concurrent.Future
+
+import akka.Done
+import akka.NotUsed
+
+import akka.stream.scaladsl.Flow
+import akka.stream.scaladsl.Keep
+import akka.stream.scaladsl.Sink
+import akka.stream.scaladsl.Source
+
+import slick.dbio.DBIO
+import slick.dbio.StreamingDBIO
+
+/**
+ * Methods for interacting with relational databases using Slick and akka-stream.
+ */
+object Slick {
+
+  /**
+   * Scala API: Executes a Slick query and streams the results as a Source[T, NotUsed].
+   * This works for both "typed" Slick queries and "plain SQL" queries.
+   */
+  def source[T](
+      streamingQuery: StreamingDBIO[Seq[T], T]
+  )(implicit session: SlickSession): Source[T, NotUsed] =
+    Source.fromPublisher(session.db.stream(streamingQuery))
+
+  /**
+   * Scala API:
```

Comment:
Missing comment? (because of the ':'), or is this standard?

Reply:
I added all the missing comments